fastapi_websocket_pubsub/event_broadcaster.py

```diff
@@ -78,46 +149,39 @@ async def __broadcast_notifications__(self, subscription: Subscription, data):
             data: the event data
         """
         logger.info("Broadcasting incoming event",
-                    {'topic':subscription.topic, 'notifier_id':self._id})
+                    {'topic': subscription.topic, 'notifier_id': self._id})
         note = BroadcastNotification(notifier_id=self._id, topics=[
                                      subscription.topic], data=data)
         # Publish event to broadcast
         async with self._publish_lock:
             async with self._broadcast:
                 await self._broadcast.publish(self._channel, note.json())
 
+    def _acquire_publishing_broadcast(self):
+        self._broadcast = self._broadcast_type(self._broadcast_url)
```

Comment:
1) since the acquiring is done by `async with self._broadcast`, i am not even sure self._broadcast should be a member. init a new one inside `__broadcast_notifications__ ` for every message.

2) i would rename `self._broadcast` to `broadcast_writer` to distinguish it from the other `broadcast_reader` in `__read_notifications__` and at the same time use the same terminology.

other options 
self._broadcast -> broadcast_publisher or broadcast_channel_publisher
broadcast_reader -> broadcast_listener or broadcast_channel_listener

Reply:
1. You can have multiple tasks using the same broadcaster -so you don't want each of them to init a new Broadcast() channel which might be labor intensive (and perhaps even conflict)

2.  How about `self._sharing_broadcast` and `listening_broadcast` ?

Reply:
re:2 sure, as long as the names are in the same "language" i'm cool :)